ios/Podfile.lock

```diff
@@ -19,4 +19,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ebd43b443038e611b86ede96e613bd6033c49497
 
-COCOAPODS: 1.7.1
+COCOAPODS: 1.6.0
```

Comment:
Algum motivo em particular para estarmos a usar 1.6.0?

Reply:
No reason, I didn't notice it downgraded it. 
I'll update the PR with the change

Reply:
Let's fix it with this https://github.com/vostpt/mobile-app/issues/59